Makefile

```diff
@@ -79,14 +79,14 @@ build: cdc
 
 build-failpoint:
 	$(FAILPOINT_ENABLE)
-	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc ./main.go
+	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc ./cmd/cdc/main.go
 	$(FAILPOINT_DISABLE)
 
 cdc:
-	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc ./main.go
+	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc ./cmd/cdc/main.go
 
 kafka_consumer:
-	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc_kafka_consumer ./kafka_consumer/main.go
+	$(GOBUILD) -ldflags '$(LDFLAGS)' -o bin/cdc_kafka_consumer ./cmd/kafka-consumer/main.go
```

Comment:
oops, I didn't notice the change here, since the `kafka-consumer` is only used for integration test and will not be published, do we still need to move it to the `cmd` directory? @hi-rustin

Reply:
Thanks for your review!

I think it's appropriate. Because it is also an executable program. I don't see a problem with putting it into cmd.

Reply:
For example: https://github.com/kubernetes/kubernetes/blob/master/cmd/linkcheck/links.go  may also be used only internally. But it is also placed in the cmd as an executable program.

Reply:
Fine, thank you